volttrontesting/platform/dbutils/test_mysqlfuncts.py

```diff
@@ -23,13 +23,10 @@
 
 
 IMAGES = [
-    "mysql:5.6.49",
-    "mysql:8.0.25"
+    "mysql:5.6",
```

Comment:
I think we need 5.7.35 and mysql:latest (which follows the 8.x versions) or you could make it 8.0.26.

https://hub.docker.com/_/mysql

Reply:
Will do. I'll go with mysql:latest to replace mysql:8.0.25; that way the tests could help reveal any compatibility issues with the most-up-to-date version of mysql.